.env.dev

```
@@ -1,6 +1,7 @@
 MAILGUN_API_KEY=
 MAILGUN_DOMAIN=
 SENDGRID_API_KEY=
+RESEND_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The RESEND_API_KEY key should go before the SENDGRID_API_KEY key

(UnorderedKey)